src/plugins/dataflow-tool/components/ui/dataflow-program-topbar.scss

```
@@ -0,0 +1,241 @@
+@import "../../../../components/vars";
+@import "../dataflow-vars";
+
+$topbar-height: 44px;
+$topbar-color: $gray-light;
+$duration-label-back: $gray-mid;
+
+.program-editor-topbar {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  // justify-content: space-between;
+  width: 100%;
+  flex: 0 0 $topbar-height;
+  background-color: $topbar-color;
+  border: solid 2px #ffffff;
+
+  .topbar-left {
+    width: 10%;
+  }
+
+  .topbar-center {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 80%;
+
+    .running-container {
+      &.countdown {
+        position: relative;
+        width: 208px;
+      }
+      .total {
+        display: flex;
+        flex-direction: column;
+        align-items: flex-start;
+        justify-content: center;
+        background-color: #bcbcbc;
+        width: 130px;
+        height: 28px;
+        border-radius: 3px;
+        font-size: 14px;
+        padding-left: 5px;
+        color: white;
+      }
+      .remaining {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: center;
+        position: absolute;
+        left: 118px;
+        width: 84px;
+        height: 28px;
+        border-radius: 3px 3px 3px 3px;
```

I realize this is just copy-paste code, but this can be simplified to 
```suggestion
        border-radius: 3px;
```